res/values-v14/dimens.xml

```diff
@@ -0,0 +1,5 @@
+<resources>
+
+    <dimen name="widget_margin">0dp</dimen>
```

Comment:
I have found an answer on [stackoverflow](http://stackoverflow.com/a/22295535/4244236) and in the [Android Documentation](https://developer.android.com/guide/topics/appwidgets/index.html#CreatingLayout). The reason is that API14+ adds a margin on its own and behaves thus differently. The name margin was used in the documentation, so I copied it.